addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties

```diff
@@ -70,6 +70,7 @@ quickstart.launch.start.option.label	= Start Page:
 quickstart.launch.start.pulldown.url	= URL (specify below)
 quickstart.launch.start.pulldown.zap	= Default ZAP Page
 quickstart.launch.start.pulldown.blank	= Blank Page
+quickstart.launch.start.url.access.error = Failed to access: {0}\nTry specify the URL directly in the browser.
```

Comment:
"specifying"?

Reply:
Also, which browser is the user supposed to try? One launched from ZAP or any? I can see people being confused by this. Maybe something like "Try specifying the URL directly in a browser to check it is accessible"?

Reply:
I thought it would be clear, ZAP just launched the browser. Would this be better "Failed to automatically open the URL in the launched browser:\n{0}\nTry specifying the URL directly." ?

Reply:
Ah - my bad, I was thinking this was when running the automated scan 😉 
I'm good with "Try specifying the URL directly in the browser." then

Reply:
Tweaked.